lib/SPIRV/libSPIRV/SPIRVType.h

```diff
@@ -191,7 +191,7 @@ class SPIRVTypeFloat : public SPIRVType {
     SPIRVCapVec CV;
     if (isTypeFloat(16)) {
       CV.push_back(CapabilityFloat16Buffer);
-      auto Extensions = getModule()->getExtension();
+      auto Extensions = getModule()->getSourceExtension();
```

Comment:
Seems like unrelated change to me, is it possible to move it into a separate PR?

Reply:
What about moving it to a separate commit?